### ISSUE TITLE
[Topic] Frontend checks for web accessibility

### ISSUE THREAD

**Issue by rianrietveld** [wpaccessibility/wp-a11y-docs#173]
URL: https://wpaccessibility.org/docs/testing/code/
Code: https://github.com/wpaccessibility/wp-a11y-docs/blob/main/docs/testing/code/index.md

Please read [How to contribute](https://wpaccessibility.org/docs/contribute/) before you start.

## Tasks

This content should be reviewed, restructured and expanded.
The links need to be checked and updated.
Some info/resources can be moved to the topics (and linked to) Standards and best practice.
Some topics are not frontend related.
